tools/netlist/include/DirectedGraph.h

```diff
@@ -79,7 +79,7 @@ class Node {
         return *this;
     }
     Node<NodeType, EdgeType>& operator=(Node<NodeType, EdgeType>&& node) noexcept {
-        edges = std::move(node.Edges);
+        edges = std::move(node.edges);
```

Comment:
What happened here?

Reply:
Syntax error:
```
third_party/sv_slang/tools/netlist/include/DirectedGraph.h:82:32: error: no member named 'Edges' in 'Node<NodeType, EdgeType>'; did you mean 'edges'?
   82 |         edges = std::move(node.Edges);
      |                                ^~~~~
      |                                edges
third_party/sv_slang/tools/netlist/include/DirectedGraph.h:156:18: note: 'edges' declared here
  156 |     EdgeListType edges;
      |                  ^
1 error generated.
Target //third_party/sv_slang:slang-netlist failed to build
```

And indeed, the only data member `Node` has is called `edges`.

Reply:
Hmm, I guess it's templated and nothing actually calls it.